client/src/Widgets/ActivityTable.tsx

```diff
@@ -0,0 +1,277 @@
+              if (newTitle !== activity.title && activity.id !== undefined) {
+                fetcher.submit(
+                  {
+                    _action: "update title",
+                    id: activity.id,
+                    cardTitle: newTitle,
+                    isFolder: Boolean(activity.isFolder),
+                  },
+                  { method: "post" },
+                );
+              }
+            }
+
+            return (
+              <Tr
+                key={(activity.authorRow ? "author" : "activity") + activity.id}
+                cursor="pointer"
+                _hover={{ backgroundColor: "#eeeeee" }}
+                borderBottom="2px solid gray"
+                onClick={() =>
+                  activity.cardLink && !titleBeingEdited
+                    ? navigate(activity.cardLink)
+                    : null
+                }
+              >
+                <Td p="0" m="0" width="20px">
+                  <Tooltip
+                    label={
+                      activity.isFolder
+                        ? activity.isPublic || activity.isShared
+                          ? "Folder / Public"
+                          : "Folder / Private"
+                        : activity.authorRow
+                          ? activity.ownerName
+                          : "Activity" +
+                            (activity.assignmentStatus
+                              ? " / " + activity.assignmentStatus
+                              : "")
+                    }
+                  >
+                    <Box paddingRight="1em" m="0">
+                      {activity.authorRow ? (
+                        <Avatar
+                          size="sm"
+                          name={activity.ownerName}
+                          marginLeft="1em"
+                        />
+                      ) : (
+                        <Icon
+                          as={
+                            activity.isFolder
+                              ? FaFolder
+                              : activity.assignmentStatus === "Unassigned"
+                                ? RiDraftFill
+                                : MdAssignment
+                          }
+                          color={
+                            activity.isFolder
+                              ? "#e6b800"
+                              : activity.assignmentStatus === "Unassigned"
+                                ? "#ff6600"
+                                : activity.assignmentStatus === "Open"
+                                  ? "#009933"
+                                  : "#cc3399"
+                          }
+                          boxSize={12}
+                          paddingLeft="1em"
+                        />
+                      )}
+                    </Box>
+                  </Tooltip>
+                </Td>
+                <Td p="0" whiteSpace="normal">
+                  <HStack>
+                    <Editable
```

Comment:
I made a PR to this branch to fix a regression I made in #2472 that broke the ability to change a content name in the `Settings` panel.

Once you merge that in, you'll see another flaw here. If you change the name using the `Settings` panel (from the three-dot menu), you'll notice that the title of the card does not update to the new name.

You can look at `<EditableName>` of `ActivityEditor.tsx` to see how someone else addressed the same issue by making the `<Editable>` a controlled component and changing its value if it was changed externally.

Also, it works for the card view, so I don't know what's different there. (It might be easier on our future selves to unify how the editable title works in both card view and list view so we don't have to maintain two approaches.)